packages/hint-manifest-icons/src/hint.ts

```diff
@@ -6,16 +6,18 @@ import { imageSize as getImageData } from 'image-size';
 import imageType from 'image-type';
 import { IHint, NetworkData, HintContext } from 'hint';
 import { JSONLocationFunction } from '@hint/utils-json';
-import { ManifestEvents, ManifestParsed, ManifestImageResource } from '@hint/parser-manifest';
+import { ManifestEvents, ManifestParsed, ManifestImageResource, ManifestInvalidSchema } from '@hint/parser-manifest';
 import { ProblemLocation, Severity } from '@hint/utils-types';
 import { debug as d } from '@hint/utils-debug';
 import { determineMediaTypeBasedOnFileExtension } from '@hint/utils/dist/src/content-type';
+import compact = require('lodash/compact');
 
 import meta from './meta';
 import { getMessage } from './i18n.import';
 import { extname } from 'path';
 
 const debug: debug.IDebugger = d(__filename);
+const iconsErrorRegex: RegExp = /\.icons\[\d\]\.(purpose)/i;
```

Comment:
Please include a comment with an example error message you expect this regex to match.
```js
// example: manifest.icons9.purpose